playgame.py

```diff
@@ -60,18 +60,19 @@ def run_tournament(args):
     disqualified = []
     for players in pairings:
         if set(players) != set(disqualified):
-            try:
-                game = HanabiGame(players, args.seed, HanabiVariant(args.variant))
-                game.play_game(args)
-                score = game.table.score()
-                for player in players:
-                    tournament_scores[player].append(score)
-            except InvalidHanabiMoveException as err:
-                disqualify_player(disqualified,
-                    tournament_scores,
-                    players[err.player_id])
-            finally:
-                rotate_logs()
+            for _ in range(args.iterations):
```

Comment:
I want to call run_one_game here, but I'd have to return the score?

Reply:
What is the downside of returning the score?

Reply:
none really...